features/facebookImport/test/name-importer.test.js

```diff
@@ -0,0 +1,66 @@
+"use strict";
+
+import { ZipFileMock } from "./mocks/zipfile-mock";
+import { runNameImporter } from "./utils/data-importing";
+import {
+    expectImportSuccess,
+    expectInvalidContentError,
+    expectMissingFileError,
+} from "./utils/importer-assertions.js";
+
+const profileInformationFileName =
```

Comment:
As @fhd has said, this should probably be spun off to a configuration file that's read from here as well as the real code.